tests/manage/z_cluster/nodes/test_simultaneous_ocs_node_drains.py

```diff
@@ -0,0 +1,142 @@
+import logging
+import pytest
+
+import time
+
+from ocs_ci.ocs.exceptions import CephHealthException
+from ocs_ci.utility.utils import ceph_health_check_base
+from ocs_ci.ocs.cluster import validate_osd_pdb
+
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.node import drain_nodes, schedule_nodes, get_nodes, get_node_objs
+from ocs_ci.framework.testlib import (
+    tier4,
+    tier4b,
+    ManageTest,
+    ignore_leftovers,
+    skipif_bm,
+    skipif_lso,
+)
+from ocs_ci.helpers.sanity_helpers import Sanity
+
+
+log = logging.getLogger(__name__)
+
+
+@pytest.fixture(autouse=True)
+def teardown(request):
+    """
+    Tear down function
+
+    """
+
+    def finalizer():
+        """
+        Make sure that all cluster's nodes are in 'Ready' state and if not,
+        change them back to 'Ready' state by marking them as schedulable
+        """
+        scheduling_disabled_nodes = [
+            n.name
+            for n in get_node_objs()
+            if n.ocp.get_resource_status(n.name)
+            == constants.NODE_READY_SCHEDULING_DISABLED
+        ]
+        if scheduling_disabled_nodes:
+            schedule_nodes(scheduling_disabled_nodes)
+
+    request.addfinalizer(finalizer)
+
+
+@ignore_leftovers
+class TestSimultaneousOcsNodeDrains(ManageTest):
+    """"""
+
+    @pytest.fixture(autouse=True)
+    def init_sanity(self):
+        """
+        Initialize Sanity instance
+
+        """
+        self.sanity_helpers = Sanity()
+
+    @pytest.fixture(autouse=True)
+    def health_checker(self):
+        """
+        Check Ceph health
+
+        """
+        try:
+            status = ceph_health_check_base()
+            if status:
```

Comment:
you can change:
try:
    ceph_health_check_base()
except CephHealthException as e:
    # skip because ceph is not in good health
    pytest.skip(str(e))